doc/whats-new.rst

```diff
@@ -173,6 +175,11 @@ Bug fixes
   ``reset_coords(drop=True)`` should now be unnecessary in most cases.
   (:issue:`3068`, :issue:`3311`, :issue:`3454`, :pull:`3453`).
   By `Deepak Cherian <https://github.com/dcherian>`_.
```

Comment:
bad merge?

Reply:
Yes. Some issues with merge. I added my comments to the wrong version section, merged with upstream, then moved it to the correct one. But I have no idea how to fix this.

Reply:
the easiest way I know would be to redo the merge: create a backup branch, hard reset to `HEAD~2`. Then I'd move the `whats-new.rst` (by cherry-picking?) and redo the merge (which you could also do by rebasing on master). Post here if you need any help with that.

Also, it seems you have a mismatch in email addresses between the one used for github and the one in the commits.